File: PythonPatternPrograms/Symbol Patterns/Pattern 78.py
for row in range(0, 6):
    for col in range(0, 7):
        if (row == 0 and col % 3 != 0) or (row == 1 and col % 3 == 0) or (row - col == 2)or(row + col == 8):
            print("*",end=" ")
        else:
            print(" ",end=" ")
    print(" ")

File: CSharpPatternPrograms/SymbolPatterns/Pattern 78.cs
using System;

class HeartPattern
{
    public static void Main()
    {
        for (int row = 0; row < 6; row++)
        {
            for (int col = 0; col < 7; col++)
            {
                if ((row == 0 && col % 3 != 0) || (row == 1 && col % 3 == 0) || (row - col == 2) || (row + col == 8))
                {
                    Console.Write("*");
                }
                else
                {
                    Console.Write(" "); // one space
                }
            }
            Console.WriteLine("");
        }
        Console.ReadKey(true);
    }
}
